src/controllers/forecastController.mjs

```diff
@@ -0,0 +1,27 @@
+import { getForecastFromAPI } from "../api/forecastAPI.mjs";
+import { loadCities } from "./citiesController.mjs";
+import { appendFileAsync } from "../utils/fileUtils.mjs";
+import { BASEDIR } from "../../config.mjs";
+
+const getWeatherInCities = async (citiesArr) => {
+  return await Promise.all(
+    citiesArr.map((cityObj) => getForecastFromAPI(citiesArr, cityObj.name))
+  );
+};
+
+export const getForecastForCityIes = async (city) => {
+  const citiesArr = await loadCities();
+  if (city && !citiesArr.find((cityObj) => cityObj.name === city)) {
```

Comment:
I saw the logic with `citiesArr.find((cityObj) => cityObj.name === city)` in your service (api folder for now)
So, I suggest to create a variable with that find and re-use it to provide founded city name into your forecast service

This way, we don't need to manage this question on service level and we good to throw an error right from controller level if something wrong